_pkgdown.yml

```diff
@@ -25,23 +33,64 @@ reference:
   - starts_with("wb_get_")
   - starts_with("wb_set_")
 
+- title: wb Misc
+  contents:
+  - starts_with("wb_protect")
+  - wb_hyperlink
+  - cleanup
+  - wb_freeze_pane
+  - wb_open
+  - starts_with("wb_clone")
+  - starts_with("wb_copy")
+  - wb_color
+  - wb_data
+  - wb_grid_lines
+  - wb_update_table
+  - wb_copy_cells
+
 - title: create functions
   contents:
   - starts_with("create_")
 
 - title: XML functions
-  desc: A curated list of XML functions used in openxlsx2.
+  desc: >
+    A curated list of XML functions used in openxlsx2.
   contents:
   - read_xml
   - starts_with("xml_")
   - as_xml
+  - starts_with("print")
 
-- title: All functions
+- title: Transformers
```

Comment:
is there some consensus in other packages, it somehow sounds strange, but maybe mostly due to the fact that I'm a German kid from the 80s and have to think about Optimus Prime reading transformers 🤖

Reply:
I'd prefer the second one